doc/ref_model/chapters/chapter03.rst

```diff
@@ -357,10 +357,53 @@ The hardware infrastructure manager needs to support the following functional as
+
+There are several opensource implementations of Redfish Clients and Redfish Services. 
+A refish resource aggregator could implement a scalable infrastructure manager which aggregates and exposes a standards-based Redfish interface and services northbound to an Orchestrator. 
+
+DMTF specifies the Redfish interface and model, seeking to expand the manageability domains. The prescription of what subset of the 
+Redfish model needs to be implemented for a specific manageability domain is left to other standards bodies.  Redfish had defined a 
+JSON syntax for the prescription call a Redfish Profile and implemented an application which reads the Redfish Profile and tests for 
+conformance of an implementation `Interop Validator <https://github.com/DMTF/Redfish-Interop-Validator>`__ .  The Redfish Profile mechanism is being utilized by the Open Compute Project 
+and Open Process Automation Forum.
+
+Redfish fulfills the following requirements stated in the above HW Infrastructure Manager section:
+
+-	**API/UI**: Redfish exactly specifies a programming interface.  The HTTP interface is accessible remotely via IP and locally via the Ethernet loopback mechanism.
+
+-	**Discovery**: The Redfish schema provides a physical hardware resource model including relevant information
+
+-	**Topology**: The Redfish schema contains a cable model
+
+-	**Equipment**: The Redfish schema contains a physical model and a functional model. The physical model expresses the chassis and containers and interconnects between them and environmental control functions.  The functional model expresses the logical aspects and includes configuration and  firmware status.   Both the physical and functional model have their own health/fault status information
+
+-	**Resource Allocation and Composition**: The Redfish schema has a composition model through which a client can compose a logical resource by allocating physical resources. `Redfish Composition Whitepaper <https://www.dmtf.org/sites/default/files/standards/documents/DSP2050_1.2.0_1.pdf>`__
+
+-	**Underlay Network Resources Manager**: The Redfish schema has models for fabrics, switches and SmartNICs.
+
+-	**Monitoring**: The Redfish schema contains event model for the client to receive hardware events and telemetry model for collecting information across the entire model (physical and functional) `Redfish Telemetry Whitepaper <https://www.dmtf.org/sites/default/files/standards/documents/DSP2051_1.0.0.pdf>`__
+
+-	**Additional Management Functions**: The Redfish schema has models for access management and identity management. Redfish has published work-in-progress extension for support policy management and OS/VM management `Policy Model Proposal, Redfish for Operating Systems, Applications and Containers <https://www.dmtf.org/documents/redfish-spmf/redfish-operating-systems-applications-and-containers-05wip>`__
```

Comment:
this link isn't publicly readable (access - denied).
Would you mind replacing it or worse making an linkcheck exception in setup.py